pkg/networkmanager/ip-settings.jsx

```diff
@@ -130,7 +130,25 @@ export const IpSettingsDialog = ({ topic, connection, dev, settings }) => {
 
         return false;
     };
-
+    const addressToMask = (address) => {
+        const addSplited = address.split('.');
+        if (addSplited.length !== 4)
+            return "";
+        if (addSplited[0] >= 0 && addSplited[0] <= 127) {
+            return "255.0.0.0";
+        } else if (addSplited[0] >= 128 && addSplited[0] <= 191) {
+            return "255.255.0.0";
+        } else if (addSplited[0] <= 192 && addSplited[0] <= 223) {
+            return "255.255.255.0";
+        }
```

Comment:
This will return `undefined` for addresses >= 224. React is cool with that, but it looks asymmetric here. Perhaps an `else { return null }` to clarify this? (Also see above)

Reply:
ack